lib/pliny/sidekiq/job_logger.rb

```diff
@@ -1,5 +1,9 @@
 module Pliny::Sidekiq
   class JobLogger
+    def prepare(job)
+      yield
```

Comment:
Looking at how [`sidekiq` uses `prepare`](https://github.com/mperham/sidekiq/blob/master/lib/sidekiq/processor.rb#L123-L143), shouldn't we pass the arg `job` along to yield? This only affects Rails 5 it seems.

```suggestion
      yield job
```